iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/region/AddRegionPeerProcedure.java

```diff
@@ -132,31 +143,39 @@ protected Flow executeFromState(ConfigNodeProcedureEnv env, AddRegionPeerState s
         case UPDATE_REGION_LOCATION_CACHE:
           handler.updateRegionCache(consensusGroupId, destDataNode, RegionStatus.Running);
           setKillPoint(state);
-          LOGGER.info("AddRegionPeer state {} complete", state);
+          LOGGER.info("[pid{}][AddRegion] state {} complete", getProcId(), state);
           LOGGER.info(
-              "AddRegionPeerProcedure success, region {} has been added to DataNode {}",
+              "[pid{}][AddRegion] success, region {} has been added to DataNode {}. Procedure took {} (start at {}).",
+              getProcId(),
               consensusGroupId.getId(),
-              destDataNode.getDataNodeId());
+              destDataNode.getDataNodeId(),
+              CommonDateTimeUtils.convertMillisecondToDurationStr(
+                  System.currentTimeMillis() - getSubmittedTime()),
+              DateTimeUtils.convertLongToDate(getSubmittedTime()));
```

Comment:
Using `DateTimeUtils.convertLongToDate(long timestamp, String sourcePrecision)` instead?
`DateTimeUtils.convertLongToDate(long timestamp)` will use `ns` as the precision when timestamp_precision is `ns`.

Reply:
good point